kpi/index.adoc

```diff
@@ -22,7 +22,7 @@
 |[big]*World Meteorological Organization*
 |Date: {docdate}
 |Version: {version}
-|Document location: https://wmo-im.github.io/wcmp2/kpi/wcmp2-kpi-DRAFT.html
+|Document location: https://community.wmo.int/wis-metadata-kpis
```

Comment:
https://community.wmo.int/en/activity-areas/wis/wis-metadata-kpis talks explains KPIs for WCMP v1 -- I think we should refer to the GH doc instead.

Reply:
ignore, I was reviewing an older diff

Reply:
Not sure what is going on... but my original question remains.